packages/scandipwa/src/component/ProductActions/ProductActions.component.js

```diff
@@ -101,7 +101,10 @@ export class ProductActions extends PureComponent {
     }
 
     renderStock(stockStatus) {
-        const stockStatusLabel = stockStatus === OUT_OF_STOCK ? __('Out of stock') : __('In stock');
+        const stockStatusLabel = stockStatus === OUT_OF_STOCK || !stockStatus
+            ? __('Out of stock')
+            : __('In stock');
+
```

Comment:
1. But wouldn't it better to invert the condition? If `stockStatus === IN_STOCK`, then product is in stock, while everything else is out of stock?
2. Please find all `OUT_OF_STOCK` occurrences and fix them is they need this check.